vulcanexus.repos

```diff
@@ -1,53 +1,57 @@
-    ros2/rmw_fastrtps:
-        type: git
-        url: https://github.com/eProsima/rmw_fastrtps.git
-        version: vulcanexus-galactic
-    ros2/rosidl_typesupport_fastrtps:
-        type: git
-        url: https://github.com/ros2/rosidl_typesupport_fastrtps.git
-        version: galactic
-    eProsima/fastdds_statistics_backend:
-        type: git
-        url: https://github.com/eProsima/Fast-DDS-statistics-backend.git
-        version: v0.4.0
-    eProsima/fastddsgen:
-        type: git
-        url: https://github.com/eProsima/Fast-DDS-Gen.git
-        version: v2.1.0
-    eProsima/fastddsgen/thirdparty/idl-parser:
-        type: git
-        url: https://github.com/eProsima/IDL-Parser.git
-        version: v1.1.0
-    eProsima/fastdds_monitor:
-        type: git
-        url: https://github.com/eProsima/Fast-DDS-monitor.git
-        version: v1.1.0
-    eProsima/shapes-demo:
-        type: git
-        url: https://github.com/eProsima/ShapesDemo.git
-        version: v2.5.0
-    eProsima/micro_ros_setup:
-        type: git
-        url: https://github.com/micro-ROS/micro_ros_setup.git
-        version: galactic
-    eProsima/dds_router:
-        type: git
-        url: https://github.com/eProsima/DDS-Router.git
-        version: v0.1.0
-    eProsima/vulcanexus_base:
-        type: git
-        url: https://github.com/eProsima/vulcanexus.git
-        version: galactic
+  eProsima/Fast-CDR:
+    type: git
+    url: https://github.com/eProsima/Fast-CDR.git
+    version: v1.0.24
+  eProsima/Fast-DDS:
+    type: git
+    url: https://github.com/eProsima/Fast-DDS.git
+    version: v2.6.0
+  eProsima/Fast-DDS-Python:
+    type: git
+    url: https://github.com/eProsima/Fast-DDS-python.git
+    version: 8b8f512f7a233f0c9915347e32b5474972e172cd
+  eProsima/dds_router:
+    type: git
+    url: https://github.com/eProsima/DDS-Router.git
+    version: v0.3.0
+  eProsima/fastdds_monitor:
+    type: git
+    url: https://github.com/eProsima/Fast-DDS-monitor.git
+    version: v1.1.0
+  eProsima/fastdds_statistics_backend:
+    type: git
+    url: https://github.com/eProsima/Fast-DDS-statistics-backend.git
+    version: v0.6.0
+  eProsima/fastddsgen:
+    type: git
+    url: https://github.com/eProsima/Fast-DDS-Gen.git
+    version: v2.1.2
+  eProsima/foonathan_memory_vendor:
+    type: git
+    url: https://github.com/eProsima/foonathan_memory_vendor.git
+    version: v1.2.1
+  eProsima/micro_ros_agent:
+    type: git
+    url: https://github.com/micro-ROS/micro-ROS-Agent.git
+    version: 2.0.1
+  eProsima/micro_ros_setup:
+    type: git
+    url: https://github.com/micro-ROS/micro_ros_setup.git
+    version: c607bd1564fd39311a574a509e7f3c53367b4c44
+  eProsima/shapes-demo:
+    type: git
+    url: https://github.com/eProsima/ShapesDemo.git
+    version: v2.6.0
+  eProsima/vulcanexus_base:
+    type: git
+    url: https://github.com/eProsima/vulcanexus.git
+    version: be93dd7906dfef09dc10fe75376ddee60c3e001e
+  ros2/rmw_fastrtps:
```

Comment:
Are we sure that we want to have this package inside the `ros2` folder? It is not the same RMW as the one provided by ROS 2...

Reply:
Moved it to eProsima folder.